packages/blocks/src/paragraph-block/paragraph-block.ts

```diff
@@ -83,7 +83,7 @@ export class ParagraphBlockComponent extends BlockComponent<
 
     return html`
       <div class="affine-paragraph-block-container">
-        <div class="affine-paragraph-rich-text-wrapper ${type}">
+        <div class="affine-paragraph-rich-text-wrapper ${type$.value}">
```

Comment:
Is `type$.value` not same as `type` if I want to make render reactive?

Reply:
type$.value will be automatically subscirbed by [SignalWatcher](https://www.npmjs.com/package/@lit-labs/preact-signals#signalwatcher). So you won't need to subscribe to `model.propsUpdated` slot. That's the difference.

Reply:
So my understanding is that although the data of `model.prop`, `model.prop$`, and `model.Y` are synchronized, using model.prop directly in render does not allow for reactive updates. This is because `model.prop` is not a getter of  `model.prop$.value`.

Reply:
Yep, model.prop won't be a reactive value and you'll need to re-get it manually when props updated. However, things will be different if `model.prop` is an `object` or `array`, for example:

```ts
interface ModelStyle {
  light: {
    color: string
  },
  dark: {
    color: string
  }
}

// This will trigger a update and the result will be synced to `model.$style` and yjs data
model.style.light.color = 'black';
```